Azure-Samples/azure-opensource-labs
# Adding option for deploying tailscale-private

## Purpose

* Adding a new `tailscale-private` cloudInit option to allow users to deploy VM with tailscale without a public IP or exposing ports to the internet (except for 41641 for tailscale). 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Get the code and deploy VM using Option 4 in the README.md file
* Verify you can SSH into the VM

## What to Check
Verify that the following are valid
* Verify there is no public IP resource in your resource group
* Verify UDP port 41641 is only port open to the internet

## Other Information
